soh/soh/Enhancements/randomizer/draw.cpp

```diff
@@ -10,9 +10,14 @@
 #include "objects/object_gi_hearts/object_gi_hearts.h"
 #include "objects/gameplay_field_keep/gameplay_field_keep.h"
 
+extern "C" u8 Randomizer_GetSettingValue(RandomizerSettingKey randoSettingKey);
+
 extern "C" void Randomizer_DrawSmallKey(PlayState* play, GetItemEntry* getItemEntry) {
     s32 pad;
-    s8 isColoredKeysEnabled = CVarGetInteger("gRandoMatchKeyColors", 1);
+    s8 keysanityEnabled = getItemEntry->getItemId == RG_GERUDO_FORTRESS_SMALL_KEY ?
+        Randomizer_GetSettingValue(RSK_GERUDO_KEYS) != RO_GERUDO_KEYS_VANILLA :
+        Randomizer_GetSettingValue(RSK_KEYSANITY) >= RO_DUNGEON_ITEM_LOC_ANY_DUNGEON;
```

Comment:
it's less fancy but this feels readable
```suggestion
    s8 keysanityEnabled = false;
    if (getItemEntry->getItemId == RG_GERUDO_FORTRESS_SMALL_KEY) {
        keysanityEnabled = Randomizer_GetSettingValue(RSK_GERUDO_KEYS) != RO_GERUDO_KEYS_VANILLA;
    } else {
        keysanityEnabled =
            Randomizer_GetSettingValue(RSK_KEYSANITY) ==  RO_DUNGEON_ITEM_LOC_ANY_DUNGEON ||
            Randomizer_GetSettingValue(RSK_KEYSANITY) ==  RO_DUNGEON_ITEM_LOC_OVERWORLD ||
            Randomizer_GetSettingValue(RSK_KEYSANITY) ==  RO_DUNGEON_ITEM_LOC_ANYWHERE;
    }
```

Reply:
>  also feel like knowing all the RSK values for which something is true when reading the lines can be nice

I think the new macro naming might alleviate this, by adding more context, as without it I'm left wondering "Why do we care if it's these N options?"